src/VisualStudio/Core/Def/Remote/VisualStudioRemoteHostClientProvider.cs

```diff
@@ -103,7 +103,7 @@ public IWorkspaceService CreateService(HostWorkspaceServices workspaceServices)
                 var serviceBroker = brokeredServiceContainer.GetFullAccessServiceBroker();
 
                 var configuration =
-                    (_globalOptions.GetOption(RemoteHostOptions.OOPCoreClrFeatureFlag) ? RemoteProcessConfiguration.Core : 0) |
+                    RemoteProcessConfiguration.Core |
```

Comment:
Do we still need RemoteHostOptions.OOPCoreClrFeatureFlag if we are not using it here?

Reply:
this is included for rps validation (it's off by default in RPS runs). We are not planning to remove the experiment as part of this crossgen change

Reply:
Removed